models/org_team.go

```diff
@@ -831,7 +845,10 @@ func UpdateTeamUnits(team *Team, units []TeamUnit) (err error) {
 	}
 
 	if _, err = sess.Insert(units); err != nil {
-		sess.Rollback()
+		err = sess.Rollback()
+		if err != nil {
```

Comment:
This if statement is not necessary because no matter the outcome err will be returned